modules/app/recording/styles/recording.module.css

```diff
@@ -155,4 +171,75 @@
 
 .submitButton:hover {
   background-color: #4DBCAA;
+}
+
+.modal {
```

Comment:
Can the current modal be refactor to have a resource modal with a fix style that can we rehuse?